mypy/config_parser.py

```diff
@@ -143,6 +143,11 @@ def check_follow_imports(choice: str) -> str:
     'disable_error_code': try_split,
     'enable_error_code': try_split,
     'package_root': try_split,
+    'exclude': lambda s: [
+        p.strip() for p in
+        (s.split('\n') if isinstance(s, str) else s)
+        if p.strip()
```

Comment:
I guess we can turn this into a separate function, so we can skip doing some work twice: `p.strip()` for example.

Reply:
If I were to, I'd probably also change the ini `exclude` function that I stole it from:

https://github.com/python/mypy/blob/fee25c3234d1450453a4ac7e8330abd51057129a/mypy/config_parser.py#L129

I considered using the `try_split()` function, but it would not have filtered empty strings, I think:

https://github.com/python/mypy/blob/fee25c3234d1450453a4ac7e8330abd51057129a/mypy/config_parser.py#L44-L49

Would this work better?

```python
    'exclude': list(filter(None, try_split('[\n]')))
```